src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carCostDay = 40;
+  const rentDays = days;
+
+  if (rentDays > 2) {
+    if (rentDays > 6) {
+      return carCostDay * rentDays - 50;
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item. You should create variables for main values - 20, 50 they shouldn't be just magic numbers